tests/pyfunc/test_inferred_code_path.py

```diff
@@ -0,0 +1,119 @@
+import os
+import pickle
+import sys
+
+import numpy as np
+import pytest
+import sklearn.datasets
+import sklearn.neighbors
+
+import mlflow
+from mlflow.models import Model
+
+sys.path.insert(0, ".")
```

Comment:
Didn't `monkeypatch.chdir(path)` work?

Reply:
oh, we need both `monkeypatch.chdir(path)` and `sys.path.insert(0, ".")`.

the `.` by default is not in `sys.path`

Reply:
can we use https://docs.pytest.org/en/latest/reference/reference.html#pytest.MonkeyPatch.syspath_prepend then?